xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/objects/classes/TextAreaClass.java

```diff
@@ -381,11 +381,16 @@ public void displayView(StringBuffer buffer, String name, String prefix, BaseCol
                 buffer.append(result);
             }
         } else {
-            StringBuffer result = new StringBuffer();
-            super.displayView(result, name, prefix, object, context);
+            String result;
+            BaseProperty prop = (BaseProperty) object.safeget(name);
+            if (prop != null) {
+                result = prop.toText();
```

Comment:
I guess the escaped is performed there? Could be nice to comment a bit.

Reply:
to text return a raw content, the escaping is realized by parsing and rendering the text in html using the getRenderedInlineContent method.